### PR TITLE
Update turtlecoin-nodes.json

### PR DESCRIPTION
**Pull request by bms8197**
I've created a TurtleCoin node in Bucharest, Romania, on a full gigabit internet connection and a powerful and stable server. I guess TurtleCoin fans from Romania will find this useful

### CODE REVIEW

**Comment by brandonlehmann** [comment]
Please update this PR to put your node in the correct spot in Alphabetical order by the name attribute.

File: turtlecoin-nodes.json [hunk 1]
```diff
@@ -143,6 +143,12 @@
       "url": "val.turtlenode.online",
       "port": 11898,
       "ssl": false
+    },
+    {
+      "name": "GoCrypto Romania",
+      "url": "turtlenode.gocrypto.ro",
+      "port": 11898,
+      "ssl": false
     }
```
Comment (Purehatred84) [inline turtlecoin-nodes.json]: {
      "name": "US West",
      "url": "167.99.168.158",
      "port": 11898,
      "ssl": false
    }

**Comment by brandonlehmann** [comment]
@Purehatred84 you have merge conflicts, please resolve.

**Comment by brandonlehmann** [comment]
I give up trying to fix this PR 😄 

I'll just commit the necessary change @bms8197

**Pull request closed without merging**